templates.yaml

```diff
@@ -1,38 +1,41 @@
-- name: "meet"
-  display_name: "Video conference"
-  desc: "An open source video conferencing app built on LiveKit Components, LiveKit Cloud, and Next.js"
-  url: "https://github.com/livekit-examples/meet"
-  image: "https://raw.githubusercontent.com/livekit-examples/meet/main/.github/assets/template-graphic.svg"
-  created_at: "2024-09-01T00:00:00Z"
+- name: "agent-starter-python"
+  display_name: "Python Voice Agent"
+  desc: "A starter project for Python, featuring a simple voice agent implementation"
+  url: "https://github.com/livekit-examples/agent-starter-python"
+  created_at: "2025-07-18T00:00:00Z"
+  updated_at: "2025-07-18T00:00:00Z"
+  requires: ["agent-starter-react", "agent-starter-android", "agent-starter-swift", "agent-starter-flutter", "agent-starter-react-native", "agent-starter-embed"]
   is_enabled: true
-  is_sandbox: true
-  category: "voice-and-video"
+  is_sandbox: false
+  category: "agent-starter-backend"
   tags:
-    - "nextjs"
-    - "typescript"
+    - "python"
+    - "agents"
+    - "voice"
   attrs:
-    Application: "Telepresence"
-    Type: "Frontend"
-    Tools: "Next.js, TypeScript"
+    Application: "VoiceAI, Agents"
+    Type: "Agent"
+    Tools: "Python >=3.9"
     License: "Apache-2.0"
 
-- name: "voice-assistant-frontend"
-  display_name: "Agent starter React"
-  desc: "A voice assistant showcasing realtime TTS, LLM, and STT"
+- name: "agent-starter-react"
```

Comment:
For the sandbox apps that have a frontend deployment, changing this key will break existing apps. We can do a coordinated migration at a later date, but it involves a DB migration and mass update of the cloudflare KV store.

Reply:
We need to leave as-is, along with the `requires` fields in other apps that reference this.

Reply:
yeah that makes sense - i thought that may be the case but decided to give it a try anyways